core/ecschema-metadata/src/Interfaces.ts

```diff
@@ -6,7 +6,9 @@
  * @module Metadata
  */
 
+import { BeUiEvent } from "@itwin/core-bentley";
 import { DelayedPromise } from "./DelayedPromise";
+import { SchemaItemFormatProps } from "./Deserialization/JsonProps";
```

Comment:
I believe these imports are not used anymore

Reply:
thanks, removed